### PR TITLE
Update customizable-option-interface.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Added Example of the Custom option type text field.

## Purpose of this pull request

This pull request (PR) contains the exact demo of the custom option with text field type. In the article one of the example is given but its basic example but not configured with the customizable option type case. I have added demo for the Custom option type Text field.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/graphql/interfaces/customizable-option-interface.html

### CODE REVIEW

File: src/guides/v2.4/graphql/interfaces/customizable-option-interface.md [hunk 1]
```diff
@@ -268,3 +277,74 @@ The following query returns information about the customizable options configure
   }
 }
 ```
+
+The following query returns information about the customizable options configured for the product with a `sku` of `xyz` with Custom Option type Text Field.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/interfaces/customizable-option-interface.md:281]: Need a blank line after this one and before the list to correct Linting error

File: src/guides/v2.4/graphql/interfaces/customizable-option-interface.md [hunk 2]
```diff
@@ -268,3 +277,74 @@ The following query returns information about the customizable options configure
...
+- Custom option Option Type is text field with required field.
+- Option Title is Favorite Color.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/interfaces/customizable-option-interface.md:283]:
```suggestion
- Option Title is `Favorite Color`.
```

File: src/guides/v2.4/graphql/interfaces/customizable-option-interface.md [hunk 3]
```diff
@@ -268,3 +277,74 @@ The following query returns information about the customizable options configure
...
+- Price is $5, Price Type is Fixed, Option SKU favoriteColorSku and Max. Characters is 20.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/interfaces/customizable-option-interface.md:284]:
```suggestion
- Price is `$5`, Price Type is `Fixed`, Option SKU is `favoriteColorSku` and Max. Characters is `20`.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just some formatting suggestions to make values stand out

File: src/guides/v2.4/graphql/interfaces/customizable-option-interface.md [hunk 4]
```diff
@@ -236,6 +236,15 @@ The following query returns information about the customizable options configure
           sort_order
           option_id
         }
+        ... on CustomizableFieldOption {
+            textField: value {
```
Comment (rogyar) [inline src/guides/v2.4/graphql/interfaces/customizable-option-interface.md:240]: I believe that the "textField" definition here might not be passed by the Webonyx validator (the example #2 is still correct). Also, please not that the response does not correspond the request after extending of this example.

Thanks

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by RakeshJesadiya** [comment]
@rogyar @BarnyShergold I have updated changes as per your suggestion respectively. Thank you.

File: src/guides/v2.4/graphql/interfaces/customizable-option-interface.md [hunk 5]
```diff
@@ -268,3 +268,75 @@ The following query returns information about the customizable options configure
   }
 }
 ```
+
+The following query returns information about the customizable options configured for the product with a `sku` of `xyz` with Custom Option type Text Field.
+
+- Custom option Option Type is text field with required field.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/interfaces/customizable-option-interface.md:274]: I saw that you were still getting Linting errors and I checked the rest of the file. The problem is that there is a list earlier in the file with style -
*  Text area
*  Checkbox
*  Date picker

You can only have one list style per file, so you need to change your list to use asterisks

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Need to correct linting error

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have updated the changes as per your suggestions. Thanks.

**Review by keharper — APPROVED** [review]

**Pull request merged**